poker-spirit/hud.py

```diff
@@ -12,14 +12,33 @@
 from dotenv import load_dotenv
 
 
+class PlayerStatsWindow:
+    def merge_current_and_career_stats(self, game, career_stats):
```

Comment:
seems like this method is the main thing we need in both classes? But doesn't use self. Lets make it just a function?